node_modules/oae-core/groupprofile/bundles/default.properties

```diff
@@ -0,0 +1,3 @@
+ADD_A_DESCRIPTION_TO_THIS_GROUP = Add a description to this group
+NO_DESCRIPTION_PROVIDED_YET = No description provided yet
```

Comment:
@SamPeck : When no description is available for a non-manager, do you want this message to be shown or should we just show the members?

![screen shot 2015-04-07 at 21 55 32](https://cloud.githubusercontent.com/assets/109850/7039097/57f830f4-dd71-11e4-8948-67cb8a88197e.png)